docs/source/api/regions.rst

```diff
@@ -165,3 +165,61 @@ Response Structure
 		"lastUpdated": "2018-12-06 02:14:45+00",
 		"name": "Manchester"
 	}}
+
+``DELETE``
+==========
+
+Deletes a region. If no query parameter is specified, a ``400 Bad Request`` response is returned.
```

Comment:
we should have this API marked as version added 1.5

Reply:
Marked in 2ada94e8f2.